ui/app/routes/policies/policy.js

```diff
@@ -0,0 +1,16 @@
+import Route from '@ember/routing/route';
+import withForbiddenState from 'nomad-ui/mixins/with-forbidden-state';
+import WithModelErrorHandling from 'nomad-ui/mixins/with-model-error-handling';
+import { inject as service } from '@ember/service';
+
+export default class PoliciesPolicyRoute extends Route.extend(
+  withForbiddenState,
+  WithModelErrorHandling
+) {
+  @service store;
+  model(params) {
```

Comment:
`nit`: Should say `async`. But JavaScript has us covered.

Reply:
See above note re: Ember